src/integration-test/utils/IModelsClientsTestsConfig.ts

```diff
@@ -43,6 +43,7 @@ export class IModelsClientsTestsConfig implements BaseIntegrationTestsConfig {
       },
       reporting: {
         baseUrl: process.env.APIS_REPORTING_BASE_URL ?? REPORTING_BASE_PATH,
+        groupingAndMappingBaseUrl: process.env.APIS_GROUPING_AND_MAPPING_BASE_URL ?? GROUPING_AND_MAPPING_BASE_PATH,
```

Comment:
Add a new property to the parent object - `groupingAndMapping`.

Reply:
The new property has been added